controllers/territory_controller/tests.py

```diff
@@ -216,40 +231,34 @@ def setUp(self, _: object, __: object, ___: object) -> None:
         )
 
     @patch('controller.Supervisor.getFromDef')
+    @patch('territory_controller.LOCKED_OUT_AFTER_CLAIM', new=2)
+    @patch('territory_controller.TERRITORY_LINKS', new={
+        (StationCode.PN, StationCode.EY),
+        (TerritoryRoot.z0, StationCode.PN),
+        (TerritoryRoot.z1, StationCode.PN),
+    })
     def test_territory_lockout(self, _: object) -> None:
         """
         Test a territory is locked after the correct number of claims and
         disconnected territories aren't also locked
         """
-        for i in range(LOCKED_OUT_AFTER_CLAIM):
-            # lock status is tested before capturing since the final
-            # iteration will lock the territory
-            for station in [StationCode.BE, StationCode.SZ]:  # assert BE, SZ unlocked
-                self.assertFalse(
-                    self.territory_controller._claim_log.is_locked(station),
-                    f"Territory {station.value} locked early on claim {i}",
-                )
-
-            # capture BE, SZ by alternating sides
-            if i % 2 == 0:
-                claimant = Claimant.ZONE_0
-            else:
-                claimant = Claimant.ZONE_1
-            for station in [StationCode.BE, StationCode.SZ]:
-                # when BE becomes locked SZ will be uncapturable
-                # which will prevent it from also locking
-                self.territory_controller.claim_territory(station, claimant, 0)
 
-        # assert BE locked
-        self.assertTrue(
-            self.territory_controller._claim_log.is_locked(StationCode.BE),
-            f"Territory BE failed to lock after {LOCKED_OUT_AFTER_CLAIM} claims",
-        )
-        # assert SZ unlocked
-        self.assertFalse(
-            self.territory_controller._claim_log.is_locked(StationCode.SZ),
-            f"Territory {station.value} incorrectly locked alongside station BE",
-        )
+        self.claim_territory(StationCode.PN, Claimant.ZONE_0)
+        self.claim_territory(StationCode.EY, Claimant.ZONE_0)
+
+        self.assertNotLocked(StationCode.PN, "early after first claims")
+        self.assertNotLocked(StationCode.EY, "early after first claims")
+
+        self.claim_territory(StationCode.PN, Claimant.ZONE_1)
+        self.claim_territory(StationCode.EY, Claimant.ZONE_1)
+
+        self.assertNotLocked(StationCode.PN, "early after second claims")
+        self.assertNotLocked(StationCode.EY, "early after second claims")
+
+        self.claim_territory(StationCode.PN, Claimant.ZONE_0)
+
+        self.assertLocked(StationCode.PN, "after third claim")
+        self.assertNotLocked(StationCode.EY, "after third claim")
```

Comment:
We haven't actually claimed this
```suggestion
        self.assertNotLocked(StationCode.EY, "after lock-out of PN")
```